2024-11-02 22:37:18 ERROR Elements not present after 0.5 seconds on page: https://capital.com/en-gb
2024-11-02 22:37:18 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 86, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 305, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x55adc905313a <unknown>
#1 0x55adc8b6e460 <unknown>
#2 0x55adc8bbaf96 <unknown>
#3 0x55adc8bbb221 <unknown>
#4 0x55adc8c005d4 <unknown>
#5 0x55adc8bdf03d <unknown>
#6 0x55adc8bfdaf6 <unknown>
#7 0x55adc8bdedb3 <unknown>
#8 0x55adc8badc10 <unknown>
#9 0x55adc8baebee <unknown>
#10 0x55adc901f3cb <unknown>
#11 0x55adc9023368 <unknown>
#12 0x55adc900ceec <unknown>
#13 0x55adc9023ee7 <unknown>
#14 0x55adc8ff213f <unknown>
#15 0x55adc9041858 <unknown>
#16 0x55adc9041a20 <unknown>
#17 0x55adc9051fb6 <unknown>
#18 0x7f9a61e94ac3 <unknown>